.github/workflows/stale.yml

```diff
@@ -0,0 +1,33 @@
+name: Check stale issues and pull requests
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 12 * * *'
+
+permissions:
+  issues: write
+  pull-requests: write
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v5
+        with:
+          debug-only: true
+          # disable issues
+          days-before-issue-stale: -1
+          days-before-issue-close: -1
+          stale-pr-message: |
+            This pull request is stale because it has been open for 30 days with no activity.
+            It will be closed in 7 days unless the `stale` label is removed.
+          stale-pr-label: stale
+          days-before-pr-stale: 14
+          days-before-pr-close: 7
+          exempt-pr-labels: 'external contribution'
+          exempt-draft-pr: true
+          exempt-all-milestones: true
+          remove-stale-when-updated: true
+          operations-per-run: 60
+          start-date: "2022-09-14"
```

Comment:
Actually, for PRs, we don't need this - it'd be nice to have this run on all PRs including the old ones :-).